tests/plugins/test_fetch_worker.py

```diff
@@ -0,0 +1,162 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import print_function, unicode_literals
+
+import os
+
+from flexmock import flexmock
+
+from atomic_reactor.core import DockerTasker
+from atomic_reactor.constants import PLUGIN_FETCH_WORKER_METADATA_KEY
+from atomic_reactor.build import BuildResult
+from atomic_reactor.inner import DockerBuildWorkflow
+from atomic_reactor.plugin import PostBuildPluginsRunner
+from atomic_reactor.util import ImageName
+
+from atomic_reactor.plugins.build_orchestrate_build import (WorkerBuildInfo, ClusterInfo,
+                                                            OrchestrateBuildPlugin)
+
+from tests.constants import MOCK_SOURCE, TEST_IMAGE, INPUT_IMAGE
+from tests.docker_mock import mock_docker
+import pytest
+
+
+class mock_ConfigMapResponse(object):
+    def __init__(self, data):
+        self.data = data
+
+    def get_json_by_key(self, key):
+        return self.data[key]
+
+
+class mockOSBS(object):
```

Comment:
nitpick: mix of mock_X, mockX, MockX style naming. For classes, MockOSBS is the preferred style.